packages/back-end/src/util/fastly.util.ts

```diff
@@ -0,0 +1,39 @@
+import fetch from "node-fetch";
+import { logger } from "./logger";
+import { FASTLY_API_TOKEN, FASTLY_SERVICE_ID } from "./secrets";
+
+export function getSurrogateKey(
```

Comment:
Moved two of the functions dealing with the CDN to a new util file, but would love some feedback on whether or not this makes sense

Reply:
intuition is that the async purge function should probably remain in services but wasn't sure if it still made sense in `features.ts`

Reply:
A util file makes sense to me.  I might change it to be `cdn.util.ts` since we plan to eventually support other CDNs besides Fastly.